website_compassion/controllers/my_account.py

```diff
@@ -320,6 +347,9 @@ def my_donations(self, form_id=None, **kw):
             len(sponsorship.filtered(lambda s: s.type == "S"))
             for sponsorship in sponsorships_by_group
         ]
+        paid_sponsor = (partner.contracts_fully_managed
+                        + partner.contracts_correspondant) \
+            .filtered(lambda a: a.state not in ["cancelled", "terminated"])
```

Comment:
Same as above
`contracts_correspondant` refers to contracts where current partner doesn't pay. `contracts_paid` seems more appropriate